docs/release/cut-data-release.md

```diff
@@ -61,7 +67,7 @@ Day 2: Pushing the Release
 
 ### Step 1: Push from pre-release to release
 
-This script moves the packages into release, clones releases into new version-specific release repos, locks the repos and regenerates them. Afterwards, it produces `*release-note*` files that should be used to update the release note pages. Clone it from the github repo and run the script:
+This script moves the packages into release, clones releases into new version-specific release repos, locks the repos and regenerates them. Afterward, it produces `*release-note*` files that should be used to update the release note pages. Clone it from the github repo and run the script:
```

Comment:
This is fine but seems unnecessary? http://grammarist.com/usage/afterward-afterwards/